### DISCUSSION TITLE
The Relation Between Wins and Runs

### DISCUSSION THREAD

**Discussion by StuffbyYuki** [has2k1/plotnine#859] [2024 Plotnine Contest]
### Authors

Yuki Kakegawa

### Links

[linkedin](https://www.linkedin.com/in/yukikakegawa/), [consulting company](https://www.oremdata.com/), [personal website](https://yukikakegawa.me/), [medium](https://medium.com/@yukithejapanese)

### Full description

Recreations of a few visualizations found in [Chapter 4: The Relation Between Runs and Wins - Analyzing Baseball Data with R (3e)](https://beanumber.github.io/abdwr3e/04-pythagoras.html). Details are in the notebook.

### Code repository

https://github.com/StuffbyYuki/Plotnine-Contest/blob/main/2024/analyzing_baseball_data_with_r/relation_between_runs_and_wins.ipynb

**Comment by has2k1** [comment]
![download](https://github.com/user-attachments/assets/e6bf2cac-49ab-4ab7-93c6-fba661de3ff1)
<hr/>

![download (1)](https://github.com/user-attachments/assets/3eab2e3e-f054-48c3-9feb-453a942ccc06)
<hr/>

![download (2)](https://github.com/user-attachments/assets/82e6321b-bf4b-4206-817f-2682117e34b2)

**Comment by EconomiCurtis** [comment]
Ahoy @StuffbyYuki , 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@StuffbyYuki , The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
